packages/unlock-js/src/walletService.ts

```diff
@@ -868,7 +868,31 @@ export default class WalletService extends UnlockService {
   }
 
   /**
-   * Upgrade a lock to a specific version
+   * Allow a Lock manager to change the transfer fee.
+   * @param {*} params
+   * @param {*} callback
+   */
+  async updateTransferFee(
+    params: {
+      lockAddress: string
+      transferFeeBasisPoints: number
```

Comment:
should we allow values in percentage here? Seems more friendly than using basis points.

Reply:
but is stores the basis point not the percentage value, can cause confusion